projects/zxing-scanner/src/lib/zxing-scanner.component.ts

```diff
@@ -195,11 +195,11 @@ export class ZXingScannerComponent implements OnInit, OnDestroy {
 
     if (!this._ready) {
       this._devicePreStart = device;
-      // let's ignore silently, users don't liek logs
+      // let's ignore silently, users don't like logs
       return;
     }
 
-    if (this.isAutostarting) {
+    if (this.iAutostarting) {
```

Comment:
there is an "s" missing.. "isAu.." and not "iAu"..
Sorry for all the inconvenience 😅

Reply:
ahhhhh! Sorry, I should have stashed my changes and started fresh. Okay, pushed up another.